CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# v0.2.0
```

Comment:
We should specify that this is the first release of the crate as we did for the other components (especially since we are starting from 0.2.0).

Reply:
done 👍🏻